app/move/steps/create.js

```diff
@@ -108,7 +108,6 @@ module.exports = {
     controller: Assessment,
     next: 'save',
     pageTitle: 'moves::steps.health_information.heading',
-    buttonText: 'actions::continue',
```

Comment:
Putting the comment here - not necessarily applicable to this line - I think the last button in the process should say "Schedule Move", at the moment it still says "Continue"

Reply:
Ah! Good spot! I'll add this logic to the last step.